README.md

```diff
@@ -2,6 +2,23 @@
 <!-- Edit the title above with your project title -->
 
 ## Project Overview
+### Topic: Can we use ML to make predictions about real estate?
+This topic is important because it can help both businesses and consumers.
+### Project Questions:
+* Can ML predict a house price with different variables (regression)?
+* Can my mom predict house prices better than ML?
+### What would an answer look like:
+* Prediction prices in the test dataset are close to the actual price
+* Dot plot that shows model accuracy 
+* Histogram of the residuals
+* Bar chart of the ML prediction, mom's prediction, and the actual price
+### Data Sources:
+* Scraping Zillow
+* Google Maps API
+* Mapbox Satellite API
```

Comment:
what types of different data are expecting to get from each of those data sources?
Why do you need them all?
How viable is scraping Zillow?

Reply:
I would look into other data sources. such as city data, school district data.